### PR TITLE
language setting

### PR DESCRIPTION
**Pull request by sadaf895**
closes: #XX

### Visible/Frontend Changes
- [x] 
- [ ] 

### Architectural/Backend Changes
- [x] 
- [ ]

### CODE REVIEW

File: src/app/core/language/language-select/language-select.component.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ import { MatMenuModule } from "@angular/material/menu";
 import { NgForOf } from "@angular/common";
 import { LOCALE_ENUM_ID } from "../languages";
 import { ConfigurableEnumDirective } from "../../basic-datatypes/configurable-enum/configurable-enum-directive/configurable-enum.directive";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
```
Comment (sleidig) [inline src/app/core/language/language-select/language-select.component.ts:12]:
```suggestion
```

Looks like IconProp is not actually used anymore

Comment (sadaf895) [reply src/app/core/language/language-select/language-select.component.ts:12]: Ok I will remove it

File: src/app/core/language/language-select/language-select.component.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-
 <mat-menu #menu>
   <a
     *appConfigurableEnum="let lang of localeEnumId"
     (click)="changeLocale(lang.id)"
     mat-menu-item
   >
-    <span [class]="'flag-clip fi fi-' + lang.label"></span>
+    <span [class]="'fa-solid fa-user' + lang.label"></span>
+    <span [class]="lang.label"></span>
```
[the remark this answers is not in the document]
Comment (sadaf895) [reply src/app/core/language/language-select/language-select.component.html:11]: Ohh I have forgot to remove it actually previously I wanted to add a user icon in front of a language abbreviation.

**Review by sleidig — APPROVED** [review]
Thank you for contributing this, @sadaf895 ! :smiley:

**Review by TheSlimvReal — COMMENTED** [review]
Thanks for taking this up @sadaf895!

Just a small suggestion when first looking at this. I was first a bit confused by the icon as I think I have never seen it before in another application. 
I think it is more common to use the `globe` icon or just show the abbreviation of the currently selected locale.

**Pull request merged**